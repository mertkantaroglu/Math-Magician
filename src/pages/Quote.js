import '../App.css';

const Quote = () => (
  <div>
    <p className="quote">
      Mathematics is not about numbers, equations or algorithms: it is about
      understanding. - William Paul Thurston
    </p>
  </div>
);

export default Quote;
