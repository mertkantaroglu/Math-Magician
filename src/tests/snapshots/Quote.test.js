import renderer from 'react-test-renderer';
import Quote from '../../pages/Quote';

describe('Quote', () => {
  test('should match with Quote snapshot', () => {
    const tree = renderer.create(
      <Quote />,
    );
    expect(tree).toMatchSnapshot();
  });
});