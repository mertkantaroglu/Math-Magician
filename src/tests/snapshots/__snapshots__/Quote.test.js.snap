// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should match with Quote snapshot 1`] = `
<div>
  <p
    className="quote"
  >
    Mathematics is not about numbers, equations or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;
